src/app/app.module.ts

```diff
@@ -56,7 +59,10 @@ type StoreType = {
     HomeComponent,
     HeaderComponent,
     FooterComponent,
+    LoginComponent,
```

Comment:
Ogólnie to sugerowałbym stworzenie `SharedModule` gdzie wrzucałbyś wszystkie komponenty, serwisy, dyrektywy, które mogą być re-używane, dzielone np. headerComponent, footerComponent, loginService, courseService itp. Na tym wideo o Modułach gościu pokazywał przykład jak to można zaimplementować. Mogę też Ci pokazać jak ja to zrobiłem. 
A powód dlaczego to sugeruję, że im dalej w las tym Ci bardziej app.module będzie puchnął i za niedługo trudno go będzie czytać.